docs/intro/import-guide.rst

```diff
@@ -5,7 +5,9 @@ Importing Your Documentation
    :description lang=en: Import your existing technical documentation from version control into Read the Docs.
 
 
-To import a documentation repository, visit your `Read the Docs dashboard`_ and click Import_.
+To import a public documentation repository, visit your `Read the Docs dashboard`_ and click Import_. Private
+repositories are not supported.
```

Comment:
Instead of saying that private repos are not supported at all, we should link to https://docs.readthedocs.io/page/commercial/

Reply:
Good call. Great place to tell users about commercial solution. I would have found it there myself.